.circleci/config.yml

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright 2016-2018 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/openjdk:9-jdk
+        environment:
+          # Quiet Maven invoker logs (Downloading... when running /src/it)
+          MAVEN_OPTS: -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+
+      - image: circleci/mysql:5.7
```

Comment:
@adriancole do you happen to know which tests require a running MySQL instance just so I can double check on these?

Reply:
Found at least some of them and the tests are currently being skipped due to a lack of `MYSQL_USER` being set. Let me remedy that.